Makefile

```diff
@@ -9,24 +9,30 @@
 #
 
 
-CL65=cl65
+CFGDIR=/usr/local/share/cc65
+CL65=/usr/local/bin/cl65 --cfg-path $(CFGDIR)/cfg --lib-path $(CFGDIR)/lib
 AC=AppleCommander.jar
 ADDR=6000
 
 PGM=hisprite
 
-all: $(PGM)
+all: hisprite hisprite-2plus
 
 
-$(PGM):
-	@PATH=$(PATH):/usr/local/bin; $(CL65) -t apple2enh --start-addr $(ADDR) -l$(PGM).lst $(PGM).s
+hisprite:
+	$(CL65) -t apple2enh --start-addr $(ADDR) -l$(PGM).lst -o $(PGM) $(PGM).s
 	java -jar $(AC) -d $(PGM).dsk $(PGM)
 	java -jar $(AC) -p $(PGM).dsk $(PGM) BIN 0x$(ADDR) < $(PGM)
-	rm -f $(PGM)
-	rm -f $(PGM).o
-	osascript V2Make.scpt $(PROJECT_DIR) $(PGM)
+	#rm -f $(PGM)
+	#rm -f $(PGM).o
+	#osascript V2Make.scpt $(PROJECT_DIR) $(PGM)
```

Comment:
Why is the osascript commented out? This is the part that launches the emulator and inserts/runs the disk image. It's kinda the magic of this one-click build pipeline. :)